src/libNetwork/DataSender.cpp

```diff
@@ -35,8 +35,6 @@ void SendDataToLookupNodesDefault(const VectorOfNode& lookups,
   }
   LOG_MARKER();
 
-  // TODO: provide interface in P2PComm instead of repopulating the lookup into
```

Comment:
The idea was to just call `SendBroadcastMessage` directly on `lookups`.  That was before we added more logic in the for loop below, so now it is not simply copying m_lookupNodes into another vector.